src/dtclient/activegate_auth_token.go

```diff
@@ -50,6 +53,7 @@ func (dtc *dynatraceClient) createAuthTokenRequest(dynakubeName string) (*http.R
 		Name:           dynakubeName,
 		SeedToken:      false,
 		ActiveGateType: activeGateType,
+		ExpirationDate: getAuthTokenExpirationDate(),
```

Comment:
I don't see this field being used currently

I remember us talking about if we should use the creation timestamp of the secret or store the expiration date in the secret. I don't 100% remember which one we agreed on to be the better option. (I think it was the secret creation timestamp as it was easier/simpler)

However the others have a point about updating the secret would be better than just deleting it, and then you would need to use this field to determine if the contents are outdated.

Reply:
Yes, I can also remember that discussion and we came to the conclusion, that we should just delete the secret and create a new one, because managing and storing the creationTimestamp ourselves is basically the same as letting kubernetes do it. But either way it makes sense to maintain the secret and to update it instead of recreating. However, the property is currently used in the POST request to the tenant, when creating the token, so that the newly created token is only valid for 60 days